Hunted Mobile/Hunted Mobile/Hunted Mobile/Model/Location.cs

```diff
@@ -55,5 +66,9 @@ public class Location {
                 Math.Pow(location.Latitude - Latitude, 2) + Math.Pow(location.Longitude - Longitude, 2)
             );
         }
+
+        public string ToCsvString() {
```

Comment:
Heeft weinig met `csv` te maken. `ToCommaSeparatedString` is wellicht een betere naam?

Reply:
Het is in feite toch een comma separated value string?

Reply:
Klopt ja, ik zag het net even anders for some reason. Is prima zo